images/linux/scripts/installers/oc.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+################################################################################
+##  File:  aws.sh
+##  Desc:  Installs the AWS CLI
+################################################################################
+
+# Source the helpers
+source $HELPER_SCRIPTS/document.sh
+# Install the oc CLI
+curl "https://mirror.openshift.com/pub/openshift-v4/clients/oc/4.6/linux/" > oc.tar.gz
+tar xvzf oc.tar.gz
+rm oc.tar.gz
+mv oc /usr/local/bin
+
+# Validate the installation
+echo "Validate the installation"
+if ! command -v oc; then
+    echo "oc was not installed"
+    exit 1
+fi
+
+# Document the installed version
+echo "Document the installed version"
+DocumentInstalledItem "oc CLI ($(oc --version 2>&1))"
```

Comment:
`oc CLI (Error: unknown flag: --version`

Reply:
I sloppily set the wrong install link last minute. Originally I was leveraging oc 3.11 as described in the issue but modified it to 4.x.  (I have since then changed it to `latest`).
Apologies 🤦

Reply:
I mean `--version` doesn't seem to work
```
oc CLI (Error: unknown flag: --version
See 'oc --help' for usage.)
```
The string can be changed to something like this:
`DocumentInstalledItem "oc CLI $(oc version)"`
It gives proper version output:
`oc CLI Client Version: 4.5.0-202005291417-9933eb9`

Reply:
this makes me sad :( that i missed this

Reply:
and it looks like we don't need this redirection `2>&1` since nothing interesting presented in stderr 🙂

Reply:
Firstly, thanks for being so prompt with me. I hate to say but this has been off the side of my desk during my full-time gig. Just taking a look now.

Reply:
Thanks for working with me again there @miketimofeev  :) let me know if there any more issues. I think theres a direct correlation to `cupsOfCoffeeDrunk` and `carelessBugs` 🤔